components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/ApplicationAuthenticatorService.java

```diff
@@ -67,10 +67,41 @@ public static ApplicationAuthenticatorService getInstance() {
         return instance;
     }
 
+    /**
+     * This method is used to get the list of SYSTEM defined local authenticator configurations.
+     *
+     * @deprecated It is recommended to use {@link #getAllSystemDefinedLocalAuthenticators()},
+     * which returning the SYSTEM defined local application authenticator configurations.
```

Comment:
Fix the description
which returning the SYSTEM defined local application authenticator configurations -> which returns the SYSTEM defined local application authenticator configurations